drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -103,6 +103,9 @@ static inline int stm32_clock_control_on(const struct device *dev,
 	case STM32_CLOCK_BUS_AHB2:
 		LL_AHB2_GRP1_EnableClock(pclken->enr);
 		break;
+	case STM32_CLOCK_BUS_AHB3:
+		LL_AHB3_GRP1_EnableClock(pclken->enr);
+		break;
```

Comment:
commit 631cac912ad141f6442b34885474fa0356b21d62 should not reference memc, e.g. `drivers: clock_control: add ...`